### PR TITLE
Describe acceptable values for cursorBlinking

### PR DESCRIPTION
**Pull request by iporsut**
Add more description of acceptable values for cursorBlinking.

### CODE REVIEW

File: src/vs/editor/common/config/commonEditorConfig.ts [hunk 1]
```diff
@@ -602,7 +602,7 @@ const editorConfiguration: IConfigurationNode = {
 			'type': 'string',
 			'enum': ['blink', 'smooth', 'phase', 'expand', 'solid'],
 			'default': editorOptions.blinkingStyleToString(EDITOR_DEFAULTS.viewInfo.cursorBlinking),
-			'description': nls.localize('cursorBlinking', "Control the cursor animation style.")
+			'description': nls.localize('cursorBlinking', "Control the cursor animation style, accepted values are 'blink', 'smooth', 'phase', 'expand' and 'solid'")
```
Comment (V-ed) [inline src/vs/editor/common/config/commonEditorConfig.ts]: As per #53911, shouldn't these be between back ticks (`)?

> - Quotes/backticks
>     - I think that all setting names, enum values, keyboard key names, command names, etc should be surrounded by backticks and rendered as code.

**Review by V-ed — COMMENTED** [review]
As per #53911, I feel like this change would be a step back from the purpose of showing enumDescriptions in settings descriptions.

> We should add the enums and enumDescriptions to the bottom of the setting description, and audit all setting descriptions to remove the duplicated info.

In other words, there has been efforts to actually remove those kind of descriptions recently : 

- Even the `cursorStyle` setting, which had its enums listed in the description in 1.25.1, [now does not have it anymore](https://github.com/Microsoft/vscode/commit/1fad9cb846df9416ed9238c998c21ddc200a331b#diff-6728189afcd9d7e37ec62aa3e7e67056R616).

---

Basically, I feel like this PR shouldn't be merged because of the way settings descriptions will be handled in 1.26.

ping @roblourens

---

Edit :
Actually, there's a line in #54690 that says exactly what I thought :
> - An enum setting description should not duplicate info in the enumDescriptions (enumDescriptions will be rendered next to the description in both setting editors).
>     - **The description shouldn't list the possible enum values.**

**Comment by iporsut** [comment]
Thanks @V-ed for explain.
